src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -162,112 +165,123 @@ const AttendanceFormBuilder = () => {
     form.setValue("memberId", Number(memberId), { shouldValidate: true });
   };
 
-  const onSubmit = (values: AttendanceFormValues) => {
-    console.log({ values });
+  const onSubmit = async (values: AttendanceFormValues) => {
+    try {
+      const result = await onAttandanceRequset(values);
+      if (!result) {
+        form.reset();
+        toast({
+          title: "Attendance submit successfully",
+          description: "You will receive message shortly",
+        });
+      }
+    } catch (error) {
+      console.error("Submission error:", error);
+      toast({
+        title: "Something went wrong",
+      });
+      return;
+    }
   };
 
   return (
-    <Form {...form}>
-      <form
-        onSubmit={form.handleSubmit(onSubmit)}
-        className="max-w-lg mx-auto p-8 mt-10 bg-white shadow-md rounded-md space-y-6"
-      >
-        <h1 className="text-2xl font-bold text-center mb-6">Attendance Form</h1>
+    <FormWrapper>
+      <Form {...form}>
+        <form
```

Comment:
Why form within form?

Reply:
Use the <Form /> component to build our form, as it’s a requirement for working with shadcn

Reply:
But why HTML form? Isn't ShadCN's `Form` enough?